src/pages/resultsView/plots/PlotsTabSelectionHistory.ts

```diff
@@ -0,0 +1,93 @@
+import { AxisMenuSelection, PlotsTabGeneOption, PlotsTabOption } from "./PlotsTab";
+import autobind from "autobind-decorator";
+
+type Selection = {
+    gene?: PlotsTabGeneOption | undefined,
+    geneSet?: PlotsTabOption | undefined,
+    source?: PlotsTabOption | undefined,
+    treatment?: PlotsTabOption | undefined,
+
+}
+
+type FieldUpdater = (option: any) => void;
+
+export default class PlotsTabSelectionHistory {
```

Comment:
so the selection history is only per datatype? if so, then I think naming should reflect that

Reply:
also, I think `history` isnt a great name because it doesnt store indefinitely backwards, its only the most recent selection, right? so how about something like `PlotsTabLastSelectionForDatatype`. kind of a mouthful... but still might be the most clear

Reply:
Sure. I went with `LastPlotsTabSelectionForDatatype`. I think that covers your concerns.